File: Modul3/Soal 5/PRAK305-2310817210020-Muhammad_Hasbiyan_Rusyadi.py
detik =int(input(""))
jam = detik // 3600
ssdtk = detik % 3600
menit = ssdtk // 60
d = ssdtk % 60
hari = jam // 24
j = jam % 24
if hari > 0:
    print(f"{hari} hari {j:02d}:{menit:02d}:{d:02d}")
else:
    print(f"{j:02d}:{menit:02d}:{d:02d}")

File: Modul3/Soal 5/PRAK305-2310817210020-Muhammad_Hasbiyan_Rusyadi.c
#include<stdio.h>
int main(){
    int dtk;
    scanf("%d", &dtk);
    int jam = dtk / 3600;
    int ssdtk = dtk % 3600;
    int menit = ssdtk / 60;
    int d = ssdtk % 60;
    int hari = jam / 24;
    int j = jam % 24;
    if (hari > 0) {
        printf("%d hari %02d:%02d:%02d\n", hari, j, menit, d);}
        else {
        printf("%02d:%02d:%02d\n", jam, menit, d);}}